server/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
Add .env or any other secrets - we don't want your secrets accidentally inside of images - you should still be able to mount or use .env files later - it just won't send it to the build context now